content/20.pharmaceuticals.md

```diff
@@ -2,8 +2,25 @@
 
 ### Abstract
 
+After emerging in China in late 2019, the novel _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021, continues to significantly impact most countries.
+Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.
```

Comment:
```suggestion
Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _severe acute respiratory syndrome coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome coronavirus_, which emerged in 2012.
```

Reply:
You can correct me if I'm wrong but no need for the related part I think

Reply:
I'm sorry I'm always shooting you down on this but I think this is how they want it, based on their Excel sheet of virus names! I'm so sorry that I never incorporate these changes, we'll see if my interpretation of the style guide is correct 😂

Reply:
Haha this is funny! No problem, you know I will totally forget and say it again haha